openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TestCaseRepository.java

```diff
@@ -90,10 +92,17 @@ public RestUtil.PatchResponse<TestCaseResult> patchTestCaseResults(
   @Override
   public void setFullyQualifiedName(TestCase test) {
     EntityLink entityLink = EntityLink.parse(test.getEntityLink());
-    test.setFullyQualifiedName(FullyQualifiedName.add(entityLink.getFullyQualifiedFieldValue(), test.getName()));
+    test.setFullyQualifiedName(
+        FullyQualifiedName.add(
+            entityLink.getFullyQualifiedFieldValue(), EntityInterfaceUtil.quoteName(test.getName())));
     test.setEntityFQN(entityLink.getFullyQualifiedFieldValue());
   }
 
+  @Override
```

Comment:
Do we need a separate override for this? Isn't it always hashing fullyQualifiedName?